ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.js

```diff
@@ -125,6 +142,70 @@ export default {
         },
       },
     },
+    {
+      id: 'DelubrumSav Seeker Dead Iron',
+      // Headmarkers are randomized, so use the tether instead.
+      netRegex: NetRegexes.tether({ target: 'Trinity Seeker', id: '01DB' }),
+      condition: (data, matches) => matches.source === data.me,
+      alarmText: (data, _, output) => output.earthshaker(),
+      outputStrings: {
+        earthshaker: Outputs.earthshakerOnYou,
```

Comment:
I recommend a strategy to have these players get away from the boss and spread around the avatar as the earthshaker tethers come from the avatar that spawns and alliance is at the boss. However, I think it's nicer to keep with the familiar cactbot earthshaker callout and let the player decide how to handle it.

Reply:
That's fair.  I suspect delubrum savage will have more fixed strategies, probably?  I'd be fine being a little bit more perscriptive here, and if people complain, we can adjust.

Reply:
Looks good.